kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * This file defines the API for the core Kolibri app.
+ */
+
+// module must be specified.
+// module maps a module into the API, at the specified path.
+// By default any module specified with a relative (rather than absolute)
+// path will be aliased to allow short hand references. Obviously, this is not needed for
+// modules already in node_modules.
```

Comment:
I think my documentation didn't quite keep up with my evolving spec. Updating this.